feature-policy/header-policy-fullscreen-disabled-manual.sub.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  if (window.testRunner) {
```

Comment:
We use this in Chrome's layout test runner, but do you know if it has any actual function inside of web platform tests? (I don't see much usage of it in this repo, except in a blink-imported test)